sys/include/usb/usbus/msc.h

```diff
@@ -0,0 +1,119 @@
+/*
+ * Copyright (C) 2019-2021 Mesotic SAS
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser General
+ * Public License v2.1. See the file LICENSE in the top level directory for
+ * more details.
+ *
+ *
+ */
+
+/**
+ * @ingroup     usbus_msc
+ *
+ * @{
+ *
+ * @file
+ * @brief       USBUS Mass Storage Class functions definitions
+ *
+ * @author      Dylan Laduranty <dylan.laduranty@mesotic.com>
+ */
+
+#ifndef USB_USBUS_MSC_H
+#define USB_USBUS_MSC_H
+
+#include <stdint.h>
+#include "usb/usbus.h"
+#include "usb/usbus/msc/scsi.h"
+#include "mtd.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#ifndef USBUS_MSC_EXPORTED_DEV
+#define USBUS_MSC_EXPORTED_DEV MTD_NUMOF
```

Comment:
nitpick, but that sounds like you would say which device to export here.
How about
```suggestion
#define USBUS_MSC_EXPORTED_NUMOF MTD_NUMOF
```

Reply:
Updated.